mne/channels/tests/test_interpolation.py

```diff
@@ -439,3 +439,39 @@ def test_method_str():
         raw.interpolate_bads(method="spline")
     raw.pick("eeg", exclude=())
     raw.interpolate_bads(method="spline")
+
+
+@pytest.mark.parametrize("montage_name", ["biosemi16", "standard_1020"])
+def test_interpolate_to_eeg(montage_name):
+    """Test the interpolate_to method for EEG."""
+    # Load EEG data
+    raw, _ = _load_data("eeg")
```

Comment:
I would also test that it works fine for epochs and evoked calling the container "inst" as above